bedrock/mozorg/templates/mozorg/contribute/signup.html

```diff
@@ -178,7 +178,11 @@ <h2 class="section-title">{{ _('Now tell us a little about yourself:') }}</h2>
 
       <div class="info-content" id="info-coding">
         <h3>{{ _('Coding') }}</h3>
+      {% if l10n_has_tag('fxos_removal') %}
```

Comment:
Same reasoning: we never used this page for l10n (this is [our signup](https://www.mozilla.org/fr/contribute/signup/)), and I think we're going to drop it very soon (check with Jen and Lucy?) I don't think we should have the tag, or ask to localize these strings.